workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReadersWriters/ReaderWriterBase.cs

```diff
@@ -77,6 +77,30 @@ public void Send(TComponentUpdate update)
             }
         }
 
+        /// <summary>
+        ///     Yield authority during soft handover
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        ///     Thrown if the authority is not <see cref="Improbable.Worker.Core.Authority.AuthorityLossImminent"/>
+        /// </exception>
+        public void SendAuthorityLossImminentAcknowledgement()
+        {
+            if (!VerifyNotDisposed())
```

Comment:
Again, that double negative is blechy.

Reply:
Fixed in the other PR, but also just fitting in with the existing code. This isn't a new function.